src/Mammal.java

```diff
@@ -0,0 +1,8 @@
+public class Mammal extends Animal {
```

Comment:
this class should have a constructor with mammalVariable and animalVariable as parameters (so the constructor should call the super(animalVariable) in the first line)